plugin/web/extensions/sidebar/modals/welcomescreen/index.tsx

```diff
@@ -0,0 +1,191 @@
+import welcomeScreenVideo from "@web/extensions/sidebar/core/assets/welcomeScreenVideo.png";
+// import Backdrop from "@web/extensions/sidebar/modals/welcomescreen/component/Backdrop";
+import useHooks from "@web/extensions/sidebar/modals/welcomescreen/hooks";
+import { Checkbox, Icon } from "@web/sharedComponents";
+import Video from "@web/sharedComponents/Video";
+import { styled } from "@web/theme";
+
+const WelcomeScreen: React.FC = () => {
+  const { ShowVideo, handleDontShowAgain, dontShowAgain, handleShowVideo, handleClose } =
+    useHooks();
+
+  return (
+    <Wrapper>
+      <CloseButton>
+        <Icon size={32} icon="close" onClick={handleClose} />
+      </CloseButton>
+      {!ShowVideo ? (
```

Comment:
I think the InnerWrapper should always be shown. 
Instead, can you make the VideoWrapper `display: none;` if showVideo is false. VideoWrapper will need to be positioned absolutely so it displays overtop the InnerWrapper content.

Reply:
based on discussion with red , the close button on video will return the user to the main welcome page so, I keep it as it it and handling close in video mode to return the user to the welcome page

Reply:
e8d02a5faeea4dc4732056ccaf88ca6f433e776c